etc/docker/test/extra/rucio_default.cfg

```diff
@@ -55,7 +55,7 @@ scheme = https,davs,gsiftp,root,srm,mock,file,magnet
 #hostcert = /etc/grid-security/hostcert.pem
 #hostkey = /etc/grid-security/hostkey.pem
 transfertool = fts3
-ftshosts = https://fts:8446
+#ftshosts = https://fts:8446
```

Comment:
Any reason for commenting it out rather than deleting it here?

Reply:
There's no particular reason for this, I just commented it out of consistency with the other options in the configuration.

Reply:
The other commented out options mostly seem to be currently valid options, whereas this `ftshosts` option is deprecated - I think it's best if we fully remove it for clarity